configure

```diff
@@ -388,6 +388,7 @@ msg "inspecting environment"
 
 CFG_OSTYPE=$(uname -s)
 CFG_CPUTYPE=$(uname -m)
+ENDIAN=$(printf '\1' | od -dAn)
```

Comment:
I don't know, but some exotic environments indeed doesn't have `od`; a quick Googling reveals OpenWRT and Maemo and the such. I don't know how many people are compiling Rust *on* these platforms though...